components/core/src/clp_s/Utils.cpp

```
@@ -20,6 +21,26 @@ using std::string;
 using std::string_view;
 
 namespace clp_s {
+class SimdJsonStringEscaper::Implementation {
+public:
+    void escape(std::string& destination, std::string_view const source) {
+        m_builder.clear();
+        m_builder.escape_and_append(source);
+
```

Cppcheck (2.20.0)

[style] The function 'get_curl_err' is never used.

(unusedFunction)